_posts/2017-10-13-routing-in-react-app.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: Routing in a React application
```

Comment:
I am not sure about the title...

Reply:
I think it's fine - I didn't know what it meant at the start but did by the end :)